internal/system/agent/router.go

```diff
@@ -72,24 +74,38 @@ func operationHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	switch o.Action {
-
-	// Call the appropriate internal function (respectively, to stop, start, or restart the service(s)).
 	case STOP:
-		InvokeOperation(STOP, o.Services)
-		w.WriteHeader(http.StatusOK)
-		w.Write([]byte("Done. Stopped the requested services."))
+		err := InvokeOperation(STOP, o.Services)
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			w.Write([]byte(fmt.Sprintf("Error: %s", err.Error())))
```

Comment:
Please confirm the syntax, but I would simplify this like so:
```
w.Write([]byte(fmt.Sprintf(err.Error())))
```

Reply:
I thought having a "presentation" to the user was valuable since this is going to be returned in the API. A success returns "Done", "Error:" for an error seemed appropriate. I can easily change it, however.

You don't need the `Sprintf` if there's nothing to format.

Reply:
Agreed. Having an additional, leading `Error: ` is fine (The way you currently have it, works) 👍